### PR TITLE
Basic Layout & UI

### PR DESCRIPTION
**Pull request by Dynna**

### CODE REVIEW

**Comment by ASV44** [comment]
@Dynna add gitignore to project and clean up tracked files by excluding idea specific files

File: MyApplication/app/src/main/java/com/example/myapplication/HomeActivity.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        feedList.adapter = ItemAdapter(items, this)*/
+
+    }
+
+/*    fun addItems() {
+        items.add(Post("user1", "imageUrl"))
+        items.add(Post("user2","imageUrl2"))
+        items.add(Post("user3","imageUrl3"))
+      //  items.add("https://images.unsplash.com/photo-1502912274126-08f31a73a5f0?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=729&q=80")
```
Comment (ASV44) [inline MyApplication/app/src/main/java/com/example/myapplication/HomeActivity.kt]: Why fo you need that commented code?
Remove it you can find it latter using version control.

File: MyApplication/app/src/main/java/com/example/myapplication/NewActivity.kt [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
     lateinit var backButton: Button
+    lateinit var nextButton: Button
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_new)
         title = "MyApp"
         val message = intent.getStringExtra("message_key")
         val messageTextView: TextView = findViewById(R.id.textView)
-        messageTextView.text = "$message, glad to see you here!"
+        messageTextView.text = "Dear $message, welcome to the Authentic Social Network! The purpose of this application is to serve as a way to connect individuals with other people in order to share information, ideas, messages, creativity and interests. Authentic Social Network has no likes, followers and influencers, it's a platform for network & inspiration where mental health is a priority!"
```
Comment (ASV44) [inline MyApplication/app/src/main/java/com/example/myapplication/NewActivity.kt]: Move this string to string resource xml file

File: MyApplication/app/src/main/java/com/example/myapplication/HomeActivity.kt [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import com.example.myapplication.adapter.ItemAdapter
+import com.example.myapplication.data.Datasource
+import kotlinx.android.synthetic.main.activity_home.*
+
+class HomeActivity : AppCompatActivity() {
+   // private lateinit var posts: MutableList<Post>
+  //  val items: ArrayList<Post> = ArrayList()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_home)
```
Comment (ASV44) [inline MyApplication/app/src/main/java/com/example/myapplication/HomeActivity.kt]: Change init of activity view to view binding https://developer.android.com/topic/libraries/view-binding

File: MyApplication/app/src/main/java/com/example/myapplication/adapter/ItemAdapter.kt [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        val textView: TextView = view.findViewById(R.id.item_title)
+        val imageView: ImageView = view.findViewById(R.id.item_image)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ItemViewHolder {
+        // create a new view
+        val adapterLayout = LayoutInflater.from(parent.context)
+            .inflate(R.layout.list_item, parent, false)
+
+        return ItemViewHolder(adapterLayout)
```
Comment (ASV44) [inline MyApplication/app/src/main/java/com/example/myapplication/adapter/ItemAdapter.kt]: Change init of recycler view item to view binding https://developer.android.com/topic/libraries/view-binding

File: MyApplication/app/src/main/java/com/example/myapplication/adapter/ItemAdapter.kt [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+class ItemAdapter(
+    private val context: Context,
+    private val dataset: List<Post>
+    ): RecyclerView.Adapter<ItemAdapter.ItemViewHolder>(){
+
+    class ItemViewHolder(private val view: View): RecyclerView.ViewHolder(view) {
+        val textView_2: TextView = view.findViewById(R.id.item_author)
+        val textView: TextView = view.findViewById(R.id.item_title)
+        val imageView: ImageView = view.findViewById(R.id.item_image)
```
Comment (ASV44) [inline MyApplication/app/src/main/java/com/example/myapplication/adapter/ItemAdapter.kt]: Access view from view binding and not by `findViewById`

File: MyApplication/app/src/main/java/com/example/myapplication/adapter/ItemAdapter.kt [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    override fun onBindViewHolder(holder: ItemViewHolder, position: Int) {
+        val item = dataset[position]
+        holder.textView_2.text = context.resources.getString(item.strResourceId)
+        holder.textView.text = context.resources.getString(item.stringResourceId)
+        holder.imageView.setImageResource(item.imageResourceId)
+    }
+
+    override fun getItemCount() = dataset.size
+
+}
```
Comment (ASV44) [inline MyApplication/app/src/main/java/com/example/myapplication/adapter/ItemAdapter.kt]: Add new line to the end of all files

**Review by ASV44 — CHANGES_REQUESTED** [review]

File: MyApplication/app/src/main/java/com/example/myapplication/HomeActivity.kt (hunk 1 above)
Comment (Dynna) [reply MyApplication/app/src/main/java/com/example/myapplication/HomeActivity.kt]: just forgot to remove them, thanks

File: MyApplication/app/src/main/java/com/example/myapplication/NewActivity.kt (hunk 2 above)
Comment (Dynna) [reply MyApplication/app/src/main/java/com/example/myapplication/NewActivity.kt]: moved

**Comment by Dynna** [comment]
@ASV44 resolved comments, please check one more time

**Review by ASV44 — APPROVED** [review]
@Dynna PR approved. Feel free to merge it.

**Pull request merged**